doc/rst/conf.py

```diff
@@ -81,7 +81,7 @@ def setup(app):
 
 # The full version, including alpha/beta/rc tags.
 release = '1.26.0 (pre-release)'
-# release = '1.26.0'
+release = '1.26.0'
```

Comment:
I don't think this should be uncommented (since it's still the pre-release)?

Reply:
Fixed